docs/examples/quickstart.py

```diff
@@ -37,3 +41,43 @@
 # check OC
 assert oc.data["save_head"][0] == "all"
 assert oc.data.save_head.sel(per=0) == "all"
+
+# PLOT
+# create budget reader
+bpth = Path("./quickstart_data/mymodel.bud")
+bobj = flopy.utils.CellBudgetFile(bpth, precision="double")
+
+# set specific discharge
+spdis = bobj.get_data(text="DATA-SPDIS")[0]
+
+# create head reader
+hpth = Path("./quickstart_data/mymodel.hds")
+hobj = flopy.utils.HeadFile(hpth, precision="double")
+
+# set heads
+heads = hobj.get_alldata()
+
+# create grid
+grbpth = Path("./quickstart_data/mymodel.dis.grb")
+grid = flopy.discretization.StructuredGrid.from_binary_grid_file(grbpth)
+
+# TODO: get_specific_discharge is dependent on flopy3 model
```

Comment:
So this is where an adapter would be necessary? I guess we could try `Gwf` implementing `ModelInterface` and implement whichever properties `get_specific_discharge` needs, and stub the rest of them for now.

Reply:
Yes for get_specific_discharge.  I'm not sure if it's worth pursuing now or not.  I think we need to discuss more and decide which path we want to be on.